<launch>

    <node ns="phobos_nuclear" pkg="i3dr_stereo_camera" type="split_laser_frames" name="i3dr_split_laser_frames" output="screen">
        <!--
        <param name="non_laser_namespace" value="non_laser"/>
        <param name="with_laser_namespace" value="with_laser"/>
        -->
    </node>

</launch>